pid_controller/test/test_pid_controller.cpp

```diff
@@ -0,0 +1,387 @@
+// Copyright (c) 2022, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test_pid_controller.hpp"
+
+#include <limits>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+using pid_controller::CMD_MY_ITFS;
+using pid_controller::feedforward_mode_type;
+using pid_controller::STATE_MY_ITFS;
+
+class PidControllerTest : public PidControllerFixture<TestablePidController>
```

Comment:
You already wrote a ton of tests but it would be nice to have one unit test to ensure the PID controller output is reasonable. Like, P term is 1, current error is 1, output is 1. Something simple like that.